pages/api/startHostedActivitySessionViaHostedPagesLink/[hostedPagesLinkId].ts

```diff
@@ -67,5 +68,13 @@ export default async function handler(
   const { session_id, session_url } =
     data?.startHostedActivitySessionViaHostedPagesLink
 
-  res.status(200).json({ sessionId: session_id, sessionUrl: session_url })
+  let additionalParams = ''
+  if (!isNil(activity_id)) {
+    additionalParams += `&activity_id=${activity_id}`
+  } else if (!isNil(track_id)) {
+    additionalParams += `&track_id=${track_id}`
+  }
+  const sessionUrl = `${session_url}${additionalParams}`
```

Comment:
Can we use the built-in URLSearchParams?

```suggestion
const queryParams = new URLSearchParams();

if (!isNil(activity_id)) {
  queryParams.append("activity_id", activity_id);
} else if (!isNil(track_id)) {
  queryParams.append("track_id", track_id);
}

const sessionUrl = `${session_url}?${queryParams.toString()}`;
```

Reply:
we don't want a URL-encoded string -- it'll go from this:
`https://goto.awell.health?sessionId=abcd` (what's already returned)
to this
`https://goto.awell.health?session_id=abcd%26track_id%3Dfoo`
instead of
`https://goto.awell.health?session_id=abcd&track_id=foo`